### ISSUE TITLE
keras-cntk "Using multiply operation before LSTM layer," gives error somehow

### ISSUE THREAD

**Issue by faruknane** [microsoft/CNTK#3710]
https://github.com/keras-team/keras/issues/13072

with tensorflow it works. However, with cntk it doesn't work. Can you take a look to this topic?

**Comment by delzac** [comment]
Based on what i can see in how keras does their multiply operation, it is nothing special and should not be the reason that cause the error. Most likely have something to do with the LSTM (i'm guessing). The exception raised was on a missing dynamic axis.

**Comment by delzac** [comment]
Anyway, the API for cntk is very similar to keras so you might want to use cntk directly instead.
